src/mpl/include/queue.h

```diff
@@ -29,8 +29,8 @@
  *	@(#)queue.h	8.5 (Berkeley) 8/20/94
  */
 
-#ifndef	_SYS_QUEUE_H_
```

Comment:
This and other changes to `queue.h` here should be squashed into the commit that added the file.

Reply:
This is from an outdated diff. It is not there anymore.

Reply:
Looks like it's still here...

Reply:
Yes. My bad! Fixing this..

Reply:
Fixed now.